pkgs/os-specific/linux/acpi-call/default.nix

```diff
@@ -1,43 +1,32 @@
-{ stdenv, fetchgit, fetchpatch, kernel }:
+{ stdenv, fetchFromGitHub, kernel }:
 
-stdenv.mkDerivation {
-  name = "acpi-call-${kernel.version}";
+stdenv.mkDerivation rec {
+  name = "acpi-call-2020-04-07-${kernel.version}";
```

Comment:
Why should we change the name of it?

Reply:
This is date of commit. It is part of the version number.

Reply:
I might make a new release instead soon though.